.github/workflows/test.yml

```diff
@@ -14,7 +14,6 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - ruby: "2.4"
```

Comment:
this is a major change. 2.5 is eol since 2022-03-31 as well. the question is if we want to drop it as well. thats the version that puppet 6 ships. maybe keeping it untill end of 2022 is okay.

Reply:
We should probably adopt a voxpupuli wide policy on when to drop ruby 2.5 (and 2.6).  Normally I would say it should be dropped as soon as it is EOL but it sounds like puppet 6 will be supported until Feb of 2023: https://puppet.com/docs/puppet/7/platform_lifecycle.html

I am in favor of dropping support for 2.6 on 2022-03-31 or even now since there is no major puppet version tied to it.